spec/oppiabotSpec.js

```diff
@@ -0,0 +1,85 @@
+require('dotenv').config();
+const {createRobot} = require('probot');
+const plugin = require('..');
+const apiForSheetsModule = require('../lib/apiForSheets');
+const checkMergeConflictsModule = require('../lib/checkMergeConflicts');
+const pullRequestpayload = require('../fixtures/pullRequestPayload.json');
+
+describe('Oppiabot\'s', () => {
+  let robot;
+  let github;
+
+  beforeEach(function (done) {
+    robot = createRobot();
+    plugin(robot);
+
+    github = {
+      issues: {
+        createComment: jasmine.createSpy('createComment')
+      }
+    };
+    robot.auth = () => Promise.resolve(github);
+    done();
+  });
+
+  describe('apiForSheets function', () => {
+    beforeEach(function (done) {
+      spyOn(apiForSheetsModule, 'apiForSheets').and.callThrough();
+      spyOn(apiForSheetsModule, 'authorize').and.callThrough();
+      spyOn(apiForSheetsModule, 'checkClaSheet');
+      robot.receive(pullRequestpayload);
+      done();
+    });
+
+    it('is called', () => {
```

Comment:
The pattern for these `it` blocks should follow: `should (not) ..... `

Reply:
Done, thanks!